javascript/modules/client/src/context.ts

```diff
@@ -206,7 +208,7 @@ export class Context {
    *
    * @public
    */
-  get daoFactoryAddress(): string {
+  get daoFactoryAddress(): string | undefined {
     return this.state.daoFactoryAddress || defaultState.daoFactoryAddress;
```

Comment:
What is `defaultState.daoFactoryAddress`? This value is no longer needed, right?